packages/mui-base/src/useTabs/useTabs.types.ts

```diff
@@ -0,0 +1,41 @@
+export interface UseTabsParameters {
+  /**
+   * The value of the currently selected `Tab`.
+   * If you don't want any selected `Tab`, you can set this prop to `false`.
+   */
+  value?: string | number | false;
+  /**
+   * The default value. Use when the component is not controlled.
+   */
+  defaultValue?: string | number | false;
+  /**
+   * The component orientation (layout flow direction).
+   * @default 'horizontal'
+   */
+  orientation?: 'horizontal' | 'vertical';
+  /**
+   * The direction of the text.
+   * @default 'ltr'
+   */
+  direction?: 'ltr' | 'rtl';
+  /**
+   * Callback invoked when new value is being set.
+   */
+  onChange?: (event: React.SyntheticEvent, value: number | string | boolean) => void;
+  /**
+   * If `true` the selected tab changes on focus. Otherwise it only
+   * changes on activation.
+   */
+  selectionFollowsFocus?: boolean;
+}
+
+export interface UseTabsReturnValue {
+  tabsContextValue: {
+    idPrefix: string | undefined;
+    value: string | number | false;
+    onSelected: (e: React.SyntheticEvent, newValue: string | number | false) => void;
+    orientation?: 'horizontal' | 'vertical';
+    direction: 'ltr' | 'rtl' | undefined;
+    selectionFollowsFocus: boolean | undefined;
+  };
```

Comment:
Lets use the `TabsContextValue` interface.
```suggestion
  /**
   * Returns the values to be passed to the tabs context provider.
   */
  tabsContextValue: TabsContextValue;
```